cleaners/vuze.xml

```diff
@@ -18,34 +18,91 @@
     You should have received a copy of the GNU General Public License
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+    @app Vuze
+    @url http://www.vuze.com/
+    @os Windows, Linux
+    @cleanerversion v2.0.0
+    @cleanerdate 2019-03-18.2
+    @cleanerby Andrew Ziem (2009-02-15 - 2009-10-04) & Tobias B. Besemer (2018-12-20 - 2019-03-18)
+    @tested ok v???, Windows 7
+    @testeddate ???
+    @testedby https://github.com/Tobias-B-Besemer
+    @note Linux $$ProgramFiles$$ not yet figured out! I will do this later! Tobias.
+
 -->
 <cleaner id="vuze">
-  <label>Vuze / Azureus</label>
+  <label>Vuze/Azureus</label>
   <description>File transfer client</description>
-  <option id="backup_files">
+  <var name="Profile">
+    <value os="windows">%AppData%\Azureus</value>
+    <value os="linux">~/.azureus</value>
+  </var>
+  <var name="ProgramFiles">
+    <!--
+    Windows itself changes %ProgramFiles% for 32-bit processes running in a 64-bit operating system as documented here:
+    https://docs.microsoft.com/en-us/windows/desktop/winprog64/wow64-implementation-details
+    ...so we use %SystemDrive%\Program Files\ & %SystemDrive%\Program Files (x86)\...
+    -->
+    <value os="windows">%SystemDrive%\Program Files\Vuze</value>
+    <value os="windows">%SystemDrive%\Program Files (x86)\Vuze</value>
+    <!--
+    Linux $$ProgramFiles$$ not yet figured out! I will do this later! Tobias.
+    <value os="linux">~/.azureus</value>
+    -->
+  </var>
+  <option id="backup">
     <label>Backup files</label>
     <description>Delete the backup files</description>
-    <action command="delete" search="glob" path="~/.azureus/active/*.bak"/>
-    <action command="delete" search="glob" path="$APPDATA\Azureus\active\*.bak"/>
+    <!--
+    Was: (Tobias)
+    <action command="delete" search="glob" path="$$Profile$$\*.bak"/>
+    <action command="delete" search="glob" path="$$Profile$$\active\*.bak"/>
+    <action command="delete" search="glob" path="$$Profile$$\subs\*.vuze.bak"/>
+    -->
+    <!-- Is now: (Tobias) -->
+    <!-- Delete *.bak recursive: -->
+    <action command="delete" search="walk.files" path="$$Profile$$\" regex="\.[Bb][Aa][Kk]$"/>
```

Comment:
FYI, if there are hundreds of files and folders in the profile, this might be slow.

Reply:
OK, but I should go recursive... No one must use this option, if he/she don't want...